.gitignore

```diff
@@ -0,0 +1,3 @@
+**/__pycache__/
```

Comment:
we likely want to ignore this subdirectory anywhere in the sources, so
```suggestion
__pycache__/
```

Reply:
it works fine also without; see the documentation [1]:

> A leading "\*\*" followed by a slash means match in all directories. For example, "\*\*/foo" matches file or directory "foo" anywhere, **the same as pattern "foo"**.

(emphasis mine) :)

[1] https://git-scm.com/docs/gitignore